pkg/kubelet/eviction/threshold_notifier_linux.go

```diff
@@ -73,8 +73,14 @@ func NewCgroupNotifier(path, attribute string, threshold int64) (CgroupNotifier,
 		}
 	}()
 	epfd, err = unix.EpollCreate1(unix.EPOLL_CLOEXEC)
+	// epoll_create1() may fail either because it's not implemented (in an old kernel)
```

Comment:
how old though? Do you have an example of distribution (or at least kernel) so old that kubernetes supports?
Asking because epoll_create1 dates back to kernel 2.6.27 (!!) and glibc 2.9 (https://man7.org/linux/man-pages/man2/epoll_create1.2.html) , I don't think k8s can actually run on so ancient platforms.

Reply:
What's the minimum kernel version supported by k8s?

Reply:
The information is susprisingly scarce. Roughly 3 years ago the minimum documented requirement was kernel 3.10: https://github.com/kubernetes/kubernetes/issues/7528

Reply:
Then this part of code changes can be remove from this PR.

Reply:
ok, please remove this specific change please.